code/modules/client/preferences_customizers.dm

```diff
@@ -221,3 +221,21 @@
 		if(entry.type == entry_type)
 			return entry
 	return null
+
+/datum/preferences/proc/genderize_customizer_entries()
```

Comment:
I tested it and it is. The customizer entries are not actually fully redone. I only added it because I noticed that changing from one to the other didn't update the parts.

Reply:
i'll touch it up if this pr turns out to not get denied.